kroxylicious-filters/kroxylicious-authorization/src/test/resources/scenarios/CREATE_PARTITIONS/0/baseline.yaml

```diff
@@ -0,0 +1,69 @@
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+---
+metadata:
```

Comment:
General question - how are we making these scenario files?   Are we hand crafting them, making sure the the fields present are consistent with the field level versions?

Reply:
I use the reflective `ApiMessageSampleGenerator` to generate a random example at the appropriate version. Then I usually just tweaked the topic names in the generated output so that they are approved for that RPC. From there I can copy and modify it to make further test cases.